src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -71,3 +68,11 @@ const createHash = (data: BinaryLike, secret: string): string => {
 		.update(data)
 		.digest("hex")}`;
 };
+
+const getWebhookSecret = async (uuid?: string) => {
+	if (uuid) {
+		const gitHubServerApp = await GitHubServerApp.findForUuid(uuid);
+		return gitHubServerApp && await gitHubServerApp.decrypt("webhookSecret");
+	}
+	return envVars.WEBHOOK_SECRET;
```

Comment:
Should probably do a check here as well to see if webhook secret is defined.  if not, throw error.